contrib/fwupd.spec.in

```diff
@@ -501,6 +527,45 @@ done
 %config(noreplace)%{_sysconfdir}/fwupd/remotes.d/fwupd-tests.conf
 %endif
 
+%if 0%{?qubes_packages}
+%files qubes-vm
+%FWUPD_QUBES_VM_DIR/fwupd_common_vm.py
+%FWUPD_QUBES_VM_DIR/fwupd_download_updates.py
+%FWUPD_QUBES_VM_DIR/fwupd_usbvm_validate.py
```

Comment:
I'm not sure the `FWUPD_QUBES_VM_DIR` macro adds anything tbh

Reply:
Replaced with default macros.